### PR TITLE
[zh-cn]: update the translation of Set.add()&clear()&delete()&has()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#16465

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/set/add/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Set/add
 
 {{JSRef}}
 
-如果 `Set` 对象中没有具有相同值的元素，则 **`add()`** 方法将插入一个具有指定值的新元素到 `Set` 对象中。
+{{jsxref("Set")}} 实例的 **`add()`** 方法会在该 set 中插入一个具有指定值的新元素，如果该 `Set` 对象中没有具有相同值的元素。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/add/index.md:8]: Could we translate all the word "set" if not wrapped with inline-code.

```suggestion
{{jsxref("Set")}} 实例的 **`add()`** 方法会在该集合中插入一个具有指定值的新元素，如果该 `Set` 对象中没有具有相同值的元素。
```

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/set/add/index.md:8]: set 与 map 是否要统一翻译成 集合 与 映射 呢？

Comment (yin1999) [reply files/zh-cn/web/javascript/reference/global_objects/set/add/index.md:8]: Left my +1 for this.

/cc @jasonren0403

Comment (jasonren0403) [reply files/zh-cn/web/javascript/reference/global_objects/set/add/index.md:8]: 👍 可以顺便统一一下之前页面的翻译

File: files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Set/clear
 
 {{JSRef}}
 
-**`clear()`** 方法移除 `Set` 对象中所有元素。
+{{jsxref("Set")}} 实例的 **`clear()`** 方法移除该 set 中的所有元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:8]:
```suggestion
{{jsxref("Set")}} 实例的 **`clear()`** 方法移除该集合中的所有元素。
```

File: files/zh-cn/web/javascript/reference/global_objects/set/delete/index.md [hunk 3]
```diff
@@ -5,14 +5,14 @@ slug: Web/JavaScript/Reference/Global_Objects/Set/delete
 
 {{JSRef}}
 
-**`delete()`** 方法从 `Set` 对象中删除指定的值（如果该值在 `Set` 中）。
+{{jsxref("Set")}} 实例的 **`delete()`** 方法从该 set 中删除指定值，如果该值在 set 中。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/set/delete/index.md:8]:
```suggestion
{{jsxref("Set")}} 实例的 **`delete()`** 方法从该集合中删除指定值，如果该值在集合中。
```

File: files/zh-cn/web/javascript/reference/global_objects/set/has/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Set/has
 
 {{JSRef}}
 
-**`has()`** 方法返回一个布尔值来指示对应的值是否存在于 `Set` 对象中。
+{{jsxref("Set")}} 实例的 **`has()`** 方法返回一个布尔值来指示对应的值是否存在于该 set 中。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/set/has/index.md:8]:
```suggestion
{{jsxref("Set")}} 实例的 **`has()`** 方法返回一个布尔值来指示对应的值是否存在于该集合中。
```

File: files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md [hunk 5]
```diff
@@ -32,7 +38,7 @@ console.log(mySet.has("foo")); // true
 mySet.clear();
 
 console.log(mySet.size); // 0
-console.log(mySet.has("foo")); // false
+console.log(mySet.has("bar")); // false
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:41]: I think it's better to use "foo" (not "bar")

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:41]: No problem.

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:41]: In https://github.com/mdn/content/pull/29586.

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md:41]:
```suggestion
console.log(mySet.has("foo")); // false
```

**Review by yin1999 — APPROVED** [review]
LGTM

**Pull request merged**